pyam/utils.py

```diff
@@ -236,26 +231,26 @@ def convert_r_columns(c):
         else:
             raise ValueError(f"Invalid argument for casting `{col}: {value}`")
 
-    # all lower case
-    str_cols = [c for c in df.columns if isstr(c)]
-    df.rename(columns={c: str(c).lower() for c in str_cols}, inplace=True)
+    return df
 
-    if "notes" in df.columns:  # this came from the database
-        logger.info("Ignoring notes column in dataframe")
-        df.drop(columns="notes", inplace=True)
-        col = df.columns[0]  # first column has database copyright notice
-        df = df[~df[col].str.contains("database", case=False)]
-        if "scenario" in df.columns and "model" not in df.columns:
-            # model and scenario are jammed together in RCP data
-            scen = df["scenario"]
-            df.loc[:, "model"] = scen.apply(lambda s: s.split("-")[0].strip())
-            df.loc[:, "scenario"] = scen.apply(
-                lambda s: "-".join(s.split("-")[1:]).strip()
-            )
 
-    # reset the index if meaningful entries are included there
-    if not list(df.index.names) == [None]:
-        df.reset_index(inplace=True)
+def _format_from_database(df):
+    """Post-process database results"""
+
+    logger.info("Ignoring notes column in dataframe")
```

Comment:
For consistency with other messages, e.g., the error "Duplicate rows in `data`"
```suggestion
    logger.info("Ignoring notes column in `data`")
```